### ISSUE TITLE
Dropdown keyboard nav does not work inside datagrids

### ISSUE THREAD

**Issue by TobiasPankner** [vmware-clarity/ng-clarity#1361]
## Describe the bug

Keyboard navigation (arrow up/down) does not work when the dropdown is inside a datagrid. 
Datagrid elements are focused instead.

## How to reproduce
https://stackblitz.com/edit/clarity-15-broken-dropdown-keyboard-nav-vmjztp

1. Place a _clr-dropdown_ somewhere inside of _clr-datagrid_
2. Open the dropdown menu
3. Try to navigate with arrow keys

## Expected behavior

The arrow keys should switch around between the dropdown menu items

## Versions

**Clarity version:**

- [ ] v13.x: not tested
- [x] v15.x
- [ ] Other: 

**Framework version:**
Angular 15 with Clarity 15

**Device:**

Reproducible on chrome with Angular 15

## Additional notes

Workaround directive:
https://stackblitz.com/edit/clarity-15-broken-dropdown-keyboard-nav-workaround

**Comment by kevinbuhmann** [comment]
Your putting a component with arrow key navigation inside another component with arrow key navigation. This is difficult to manage in code and is confusing to users. We would recommend finding a more concise UX that does not nest these components.
